spec/pg/encoder_spec.cr

```diff
@@ -18,11 +18,18 @@ private def test_insert_and_read(datatype, value, file = __FILE__, line = __LINE
 end
 
 describe PG::Driver, "encoder" do
-  test_insert_and_read "int4", 123
+  test_insert_and_read "boolean", true
+  test_insert_and_read "boolean", false
```

Comment:
I noticed there weren't any tests for a plain boolean column, just an array of them. Probably didn't matter, but I needed to solve for booleans before solving for arrays of booleans. 😄

Reply:
Good catch